06 Data Library/01 Forex/Timezone.html

```diff
@@ -0,0 +1,5 @@
+<p>
+OANDA data is in GMT timezone. Although many popular charting websites convert this to EST for display QuantConnect has elected to leave it in GMT and not manipulate the data. When comparing the data to external references make sure to account for the different time zones.
```

Comment:
I see UCT and GMT -- let's keep it consistent as UTC (this is what's used in the code and other, existing documentation)